docs/.vuepress/config.js

```diff
@@ -95,10 +90,6 @@ module.exports = {
             {
               title: "Forum",
               url: "https://forum.cosmos.network/c/tendermint"
-            },
-            {
-              title: "Chat",
```

Comment:
we should have some chat, this is quite bad that we are getting rid of a way for people to talk to us

Reply:
Added link to Discord: https://discordapp.com/channels/669268347736686612.